File: app/src/main/java/com/mxt/anitrend/model/entity/base/RecommendationBase.java
package com.mxt.anitrend.model.entity.base;

import com.mxt.anitrend.model.entity.group.RecyclerItem;
import com.mxt.anitrend.util.KeyUtil;

public class RecommendationBase extends RecyclerItem {
    private long id;
    private MediaBase mediaRecommendation;
    private int rating;
    private UserBase user;
    @KeyUtil.RecommendationRating
    private String userRating;

    public long getId() {
        return id;
    }

    public MediaBase getMediaRecommendation() {
        return mediaRecommendation;
    }

    public int getRating() {
        return rating;
    }

    public UserBase getUser() {
        return user;
    }

    @KeyUtil.RecommendationRating
    public String getUserRating() {
        return userRating;
    }
}

File: app/src/main/java/com/mxt/anitrend/model/entity/base/RecommendationBase.kt
package com.mxt.anitrend.model.entity.base

import com.mxt.anitrend.model.entity.group.RecyclerItem
import com.mxt.anitrend.util.KeyUtil

open class RecommendationBase : RecyclerItem() {
    var id: Long = 0
    var mediaRecommendation: MediaBase? = null
    var rating: Int = 0
    var user: UserBase? = null

    @KeyUtil.RecommendationRating
    var userRating: String? = null
}
